### PR TITLE
All station goals are unlocked regardless of gamemode

### PR DESCRIPTION
**Pull request by SomeguyManperson**
:cl:  
rscadd: Station goals are all unlocked at the command report
/:cl:

### CODE REVIEW

**Comment by Szyszkrzyneczka** [comment]
finally

**Comment by Ratvar** [comment]
Break this curse that prevents adding anything of value to cargo that's not guns!

**Comment by Nickvr628** [comment]
I like this

File: code/game/gamemodes/game_mode.dm [hunk 1]
```diff
@@ -554,7 +554,7 @@
 	if(rev)
 		rev.remove_revolutionary(TRUE)
 
-/datum/game_mode/proc/generate_station_goals()
+/datum/game_mode/proc/generate_station_goals() //YOGS: mirrored in the yogstation gamemode file, changes here won't do anything
```
Comment (ThatLing) [inline code/game/gamemodes/game_mode.dm]: you dont have to mirror anymore silly

**Review by alexkar598 — APPROVED** [review]

**Pull request merged**